client/methods/updateMessage.coffee

```diff
@@ -23,6 +23,7 @@ Meteor.methods
 
 		Tracker.nonreactive ->
 
+			message.editBy = Meteor.userId()
```

Comment:
Agreed, but I find that things left for later never get done.  There's always so much to do and small things like this will only get harder to change later.  Blah blah blah....

Fixing it will help you learn the system  as well.  

That's my 2 cents.  :smile:   

Thanks again.